apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -0,0 +1,414 @@
+        <ToggleGroup selected={platform} onChange={this.onPlatformChange}>
+          <button type="button" value={WINDOWS}>
+            <FontAwesome icon="windows" /> {i18n.windows()}
+          </button>
+          <button type="button" value={MAC}>
+            <FontAwesome icon="apple" /> {i18n.mac()}
+          </button>
+          <button type="button" value={LINUX}>
+            <FontAwesome icon="linux" /> {i18n.linux()}
+          </button>
+          <button type="button" value={CHROMEBOOK}>
+            <FontAwesome icon="chrome" /> {i18n.chromebook()}
+          </button>
+        </ToggleGroup>
+        {WINDOWS === platform && <WindowsDownloads />}
+        {MAC === platform && <MacDownloads />}
+        {LINUX === platform && <LinuxDownloads />}
+        {CHROMEBOOK === platform && <ChromebookInstructions />}
+      </div>
+    );
+  }
+}
+
+const downloadButtonStyle = {
+  minWidth: 400,
+  textAlign: 'center'
+};
+
+class WindowsDownloads extends React.Component {
+  state = {installer: null, error: null};
+
+  componentDidMount() {
+    latestWindowsInstaller()
+      .then(installer => this.setState({installer}))
+      .catch(error => this.setState({error}));
+  }
+
+  render() {
+    const {installer, error} = this.state;
+    return (
+      <div>
+        <h2>{applabI18n.makerSetupMakerAppForWindows()}</h2>
+        {!installer && !error && <FetchingLatestVersionMessage />}
+        {installer && !error && (
+          <Button
+            __useDeprecatedTag
+            text={`Download Code.org Maker App for Windows (${
+              installer.version
+            })`}
+            icon="download"
+            color={Button.ButtonColor.orange}
+            size={Button.ButtonSize.large}
+            style={downloadButtonStyle}
+            href={DOWNLOAD_PREFIX + installer.filename}
+          />
+        )}
+        {error && <FetchingLatestVersionError />}
+        <div>
+          <h4>{i18n.instructionsWithColon()}</h4>
+          <ol>
+            <li>{applabI18n.makerSetupDownloadAndInstall()}</li>
+            <li>
+              <SafeMarkdown markdown={applabI18n.makerSetupWindows7Drivers()} />
+            </li>
+            <li>{applabI18n.makerSetupSignIn()}</li>
+            <li>{applabI18n.makerSetupPlugInBoard()}</li>
+          </ol>
+        </div>
+      </div>
+    );
+  }
+}
+
+class MacDownloads extends React.Component {
+  state = {installer: null, error: null};
+
+  componentDidMount() {
+    latestMacInstaller()
+      .then(installer => this.setState({installer}))
+      .catch(error => this.setState({error}));
+  }
+
+  render() {
+    const {installer, error} = this.state;
+    return (
+      <div>
+        <h2>{applabI18n.makerSetupMakerAppForMac()}</h2>
+        {!installer && !error && <FetchingLatestVersionMessage />}
+        {installer && !error && (
+          <Button
+            __useDeprecatedTag
+            text={`Download Code.org Maker App for Mac (${installer.version})`}
```

Comment:
Would we want to make these download button strings translatable?

Reply:
Good catch.

Reply:
untranslated button text added to the i18n pipeline